### PR TITLE
Update to 10.4 "Output Structure"

### PR DESCRIPTION
**Pull request by bwicklund**
Per the following issue: https://github.com/json-schema-org/json-schema-spec/issues/824

### CODE REVIEW

**Comment by gregsdennis** [comment]
Why is this an improvement? It's not fixing anything. There's not actually a typo in the existing text.

File: jsonschema-core.xml [hunk 1]
```diff
@@ -2636,13 +2636,13 @@ https://example.com/schemas/common#/$defs/count/minimum
                     Specifically, the errors it will produce are:
                     <list>
                         <t>
-                            The second element in the "vertices" property is missing a "y" property.
+                            The second "point" object is missing a "y" property.
```
Comment (gregsdennis) [inline jsonschema-core.xml]: I would suggest just saying `The second item` and leaving out the type.

File: jsonschema-core.xml [hunk 2]
```diff
@@ -2636,13 +2636,13 @@ https://example.com/schemas/common#/$defs/count/minimum
...
                         </t>
                         <t>
-                            The second element in the "vertices" property has a disallowed "z" property.
+                            The second "point" object has a disallowed "z" property.
```
Comment (gregsdennis) [inline jsonschema-core.xml]: Ditto

File: jsonschema-core.xml [hunk 3]
```diff
@@ -2636,13 +2636,13 @@ https://example.com/schemas/common#/$defs/count/minimum
...
                         </t>
                         <t>
-                            There are only two vertices, but three are required.
+                            There are only two "point", but three are required.
```
Comment (gregsdennis) [inline jsonschema-core.xml]: `There are only two items`.

File: jsonschema-core.xml (hunk 1 above)
Comment (bwicklund) [reply jsonschema-core.xml]: How about `The second object`? A parser would know it is an object.

Comment (gregsdennis) [reply jsonschema-core.xml]: That's fine.  Like I mentioned, this portion of the text is prose, not machine output.  The reason I suggested the edit was the edit below: `There are only two "point"` sounds weird, and I don't know how to fix it, so just reword it.

Comment (bwicklund) [reply jsonschema-core.xml]: I totally understand it is prose, but it also feels like and an example of potential error output so I think it should read as real as possible.

**Comment by Relequestual** [comment]
Interesting case here.
What would happen if the `$ref` was inlined?
You would then only know what "it" was based on any provided annotations, for example `title`.
Alterntativly, use `properties` key, OR, say "at the document root".
My feeling is the correct "fix" for this would be to include the annotation.
Additionally, we could suggest a workflow for determining the subject of the error.

Later in 10.4.2, we give the BASIC output with the error "Expected at least 3 items but found 2".

Maybe the mentioned error message was simply not sanity checked.
Either way, the ISSUE is correct, but I'm not sure on the intent or the best resolution right now.

**Comment by Relequestual** [comment]
In terms of "what's the process", let's open a new issue and get it defined properly =]

**Comment by handrews** [comment]
@gregsdennis I tried really hard to get all of the example lines within the limits of the IETF text rendering format, or at least down to only giving specification build warnings instead of errors.  I'm fairly certain I changed it to "points" to save a few letters and must have missed "vertices" here.

**Comment by handrews** [comment]
@Relequestual @gregsdennis the process is to tag these things with the appropriate milestone, which would be draft-08-patch1.  Which I've been doing as appropriate since 2019-09 was published.  See also the big table at https://json-schema.org/specification-links.html which records milestones along with every other type of version thingy we use.

**Comment by Relequestual** [comment]
I think two is the magic number here...

**Pull request merged**